pkg/controllers/management/kontainerdrivermetadata/data_getter.go

```diff
@@ -91,7 +91,12 @@ func GetRKEK8sServiceOptions(k8sVersion string, svcOptionLister v3.RKEK8sService
 		key = svcOptionWindowsKey
 	}
 	val, ok := sysImage.Labels[key]
-	if !ok {
+	// It's possible that we have a k8s version with no windows svcOptions. In this case, we just warn and return nil.
+	// if we have in fact windows nodes trying to use that version, the error will show in reknodeconfig server.
+	if !ok && osType == Windows {
+		logrus.Warnf("getSvcOptions: no service-option key present for %s", k8sVersion)
```

Comment:
can we log this in debug? don't want users to be alarmed because of node config calling this very frequently 

nit: can do osType check in `if !ok` block